comfyui-truss/README.md

```diff
@@ -0,0 +1,312 @@
+
+Here is a complete example of how you make a prediction request to your truss in python:
+
+```python
+headers = {"Authorization": f"Api-Key YOUR-BASETEN-API-KEY-HERE"}
+
+sdxl_controlnet_workflow = {
+  "3": {
+    "inputs": {
+      "seed": 972197629127129,
+      "steps": 40,
+      "cfg": 7,
+      "sampler_name": "euler",
+      "scheduler": "normal",
+      "denoise": 1,
+      "model": [
+        "14",
+        0
+      ],
+      "positive": [
+        "10",
+        0
+      ],
+      "negative": [
+        "7",
+        0
+      ],
+      "latent_image": [
+        "5",
+        0
+      ]
+    },
+    "class_type": "KSampler"
+  },
+  "5": {
+    "inputs": {
+      "width": 1024,
+      "height": 1024,
+      "batch_size": 1
+    },
+    "class_type": "EmptyLatentImage"
+  },
+  "6": {
+    "inputs": {
+      "text": "{{positive_prompt}}",
+      "clip": [
+        "14",
+        1
+      ]
+    },
+    "class_type": "CLIPTextEncode"
+  },
+  "7": {
+    "inputs": {
+      "text": "{{negative_prompt}}",
+      "clip": [
+        "14",
+        1
+      ]
+    },
+    "class_type": "CLIPTextEncode"
+  },
+  "8": {
+    "inputs": {
+      "samples": [
+        "3",
+        0
+      ],
+      "vae": [
+        "14",
+        2
+      ]
+    },
+    "class_type": "VAEDecode"
+  },
+  "10": {
+    "inputs": {
+      "strength": 0.6,
+      "conditioning": [
+        "6",
+        0
+      ],
+      "control_net": [
+        "12",
+        0
+      ],
+      "image": [
+        "16",
+        0
+      ]
+    },
+    "class_type": "ControlNetApply"
+  },
+  "11": {
+    "inputs": {
+      "image": "{{controlnet_image}}",
+      "choose file to upload": "image"
+    },
+    "class_type": "LoadImage"
+  },
+  "12": {
+    "inputs": {
+      "control_net_name": "diffusers_xl_canny_full.safetensors"
+    },
+    "class_type": "ControlNetLoader"
+  },
+  "14": {
+    "inputs": {
+      "ckpt_name": "sd_xl_base_1.0.safetensors"
+    },
+    "class_type": "CheckpointLoaderSimple"
+  },
+  "15": {
+    "inputs": {
+      "images": [
+        "16",
+        0
+      ]
+    },
+    "class_type": "PreviewImage"
+  },
+  "16": {
+    "inputs": {
+      "low_threshold": 0.2,
+      "high_threshold": 0.6,
+      "image": [
+        "11",
+        0
+      ]
+    },
+    "class_type": "Canny"
+  },
+  "18": {
+    "inputs": {
+      "images": [
+        "8",
+        0
+      ]
+    },
+    "class_type": "PreviewImage"
+  }
+}
+
+values = {
+  "positive_prompt": "An igloo on a snowy day, 4k, hd",
+  "negative_prompt": "blurry, text, low quality",
+  "controlnet_image": "https://storage.googleapis.com/logos-bucket-01/baseten_logo.png"
+}
+
+data = {"json_workflow": sdxl_controlnet_workflow, "values": values}
+res = requests.post("https://app.baseten.co/model_versions/<YOUR-MODEL-ID>/predict", headers=headers, json=data)
+res = res.json()
+model_output = res.get("model_output").get("result")
```

Comment:
You'll also have a new response format with the new endpoint, so you can drop `.get("model_output")` from this